whipper/common/program.py

```diff
@@ -470,6 +471,70 @@ def getHTOA(self):
         stop = track.getIndex(1).absolute - 1
         return start, stop
 
+    def getCoverArt(self, path, cover_art, release_id):
+        """
+        Get cover art image from Cover Art Archive if user chooses to
+
+        :type  cover_art: bool
+        :param cover_art: whether to fetch cover art or not
+        :type  path: str
+        :param path: where to store the fetched image
+        :type  release_id: str
+        :param release_id: a release id (self.program.metadata.mbid)
+
+        :rtype: str or None
+        """
+        if not cover_art:
+            return
+
+        import requests
+        import whipper
+
+        image_path = os.path.join(path, 'coverart.jpg')
+        user_agent = ("whipper/%s "
+                      "( https://github.com/whipper-team/whipper )" %
+                      whipper.__version__)
```

Comment:
It seems fragile to not use `user_agent` used by main whipper. Is that unfeasible?

Reply:
@Freso , sorry but i don't really get what you mean right now. Anyway, i have made the case similar to what was done in `common/mbngs.py` by using `musicbrainzngs.set_useragent` method.